.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: Bug Report
+description: Create a report about an error to help us improve
+title: ""
```

Comment:
When I went to view the file, several problems were listed at the top. Copying them into each relevant section:

`title must be of type String and cannot be empty.` [Learn more about error 1.](https://docs.github.com/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#key-must-be-a-string)